### DISCUSSION TITLE
[05/25] 피어세션 일지

### DISCUSSION THREAD

**Discussion by GY-Jeong** [bcaitech1/p4-dkt-lastffang#10] [Peer Session]
### 협업 관련

- 사소한 커밋도 이슈에 파고 올리기
  - 직접 master branch에 올리는 것은 지양
  - 목적은 실수 방지!
- pull request는 몇명 이상 확인해야 confirm?
  -  세번째로 approval하는 사람이 commit

### 각자 한 내용 간단하게 브리핑

- @ddoyoon 

- @scarlettJin 
  
- @alinghi 
  
- @gonipark 
  
- @newspring97 

- @GY-Jeong 
#2 testId의 대분류 EDA 진행

### 제출 규칙

- desctription 내용은 wandb의 run name과 동일하게

- wandb에서 이름 정하는 규칙은 어떻게?

  description:model-001

  run:model-001

- 제출 할 때 hyperparameter를 이용해보는 것은 어떻게 생각하세요?

  기각!

### wandb 프로젝트

- `lastffang/p-stage-4` : 모든 run 데이터 저장
- `lastffang/leaderboard` : 리더보드상에 올린 run 데이터 저장

### 깃헙 알람

- 개인 dm과 연동시켜서
- @github 초대하고 /github subscribe https://github.com/bcaitech1/p4-dkt-lastffang 하면 됩니당

### 베이스라인 코드 프리뷰

- dataloader.py에서 어떤 단위로 나누나? (사람? 시험문제??)
  - seed가 다름
- trainer.py 부터 내일 시작!

**Comment by ddoyoon** [comment]
저 슬랙에 깃헙 연동할 때 개인 dm 과 연동을 어떻게 시켜야할지 몰라서, 슬랙 좌측 상단 More -> Apps -> GitHub 누르고 GitHub 과의 conversation 에서 /github subscribe https://github.com/bcaitech1/p4-dkt-lastffang 해서 우선 해결했습니다..!
너무 바보같은 삽질이지만 👀 혹시나 해서 올립니당
